ILRuntime/Runtime/Intepreter/ILIntepreter.cs

```diff
@@ -1924,7 +1926,7 @@ public object Run(ILMethod method, object instance, object[] p)
                                             int idx = (int)ip->TokenLong;
                                             var f = t.GetField(idx);
```

Comment:
I believe it is used a few lines below for `FieldType`. It'd be nice to be able to remove it though if possible.